scripts/generate_rust_analyzer.py

```diff
@@ -0,0 +1,175 @@
+#!/usr/bin/env python3
+import sys
+import os
+from os import path
+import json
+
+if (len(sys.argv) != 5):
+    print("Not enough arguments")
+    sys.exit(-1)
+
+srctree = sys.argv[1]
+objtree = sys.argv[2]
+lib_src = sys.argv[3]
+bindgen_file = sys.argv[4]
+
+crate_list = []
+
+def find(list, name):
+    pos = -1
+    for x in list:
+        pos += 1
+        if x['display_name'] == name:
+            return pos
+
```

Comment:
To avoid returning an empty value, you can do something similar to a Rust panic, e.g.:

```python
raise RuntimeError("Not found")
```

or similar outside of the loop. You can also provide more details in the error easily using an f-string (see the other comments first).